geode-core/src/test/java/org/apache/geode/cache/query/internal/types/TypeUtilsJUnitTest.java

```diff
@@ -543,13 +525,6 @@ public void comparingUnequalPdxStringToStringShouldNotMatch() throws Exception {
 
   @Test
   public void comparingTemporalValuesShouldDelegateToTemporalComparator() {
```

Comment:
I think this and the other modified tests can be renamed now since they no longer assert that specific comparators are being used.  Instead it's just asserting that we can make comparisons against booleans, different numeric values and date/time objects